app/components/Application/ApplicationOverrideNotificationCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Card} from 'react-bootstrap';
+
+interface Props {
+  overrideJustification: string;
+}
+
+const ApplicationOverrideNotification: React.FunctionComponent<Props> = ({
+  overrideJustification
+}) => {
+  return (
+    <Card style={{margin: '1rem 0'}}>
+      <Card.Header className="bg-danger">
+        <h4>Override is active</h4>
+      </Card.Header>
+      <Card.Body>
+        <Card.Text>
+          <strong>
+            There were errors in this application that the reporter has chosen
+            to override. Justification for overriding the errors is provided in
+            the box below:
+          </strong>
+        </Card.Text>
+        <Card>
+          <div tabIndex={0} id="justification" className="scrollbar">
+            <Card.Body>{overrideJustification}</Card.Body>
+          </div>
+        </Card>
+      </Card.Body>
+      <style jsx>{`
+        #justification {
+          max-height: 19.2em;
+          background: #eee;
+          overflow-y: scroll;
+        }
+        .scrollbar::-webkit-scrollbar {
+          -webkit-appearance: none;
+          width: 8px;
+        }
+        .scrollbar::-webkit-scrollbar-thumb {
+          border-radius: 4px;
+          background-color: rgba(0, 0, 0, 0.5);
+          box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
```

Comment:
I think this may have been borrowed from the legal text disclaimer blurb shown just before certifying an application. 

It's specific to account for Chrome on MacOS, which hides scrollbars while the user is not scrolling - ignoring the CSS `overflow` setting. The UX issue it solves is that the hiding of scrollbars doesn't let the user see that there's additional content to scroll.

The MDN note would be pertinent to an expectation that it be a cross-platform feature, however in this case it's safe to target the above and at worst, is simply ignored by other browsers (which respect the overflow settings causing persistent scrollbars to be shown, as desired).

Reply:
Without the scrollbar CSS, here's what it looks like on MacOS:
<img width="1116" alt="Screen Shot 2020-07-24 at 2 37 51 PM" src="https://user-images.githubusercontent.com/5522075/88437406-4517fd80-cdbb-11ea-9eeb-4375e6c6062b.png">

With the scrollbar CSS:
<img width="1122" alt="Screen Shot 2020-07-24 at 2 40 02 PM" src="https://user-images.githubusercontent.com/5522075/88437614-afc93900-cdbb-11ea-9e4d-f638c4ff1d75.png">

Reply:
I suppose that can stay. I'd argue that the user chose that poor UX by using MacOS though. :upside_down_face: